routingpy/routers/google.py

```diff
@@ -291,6 +292,52 @@ def _parse_direction_json(response, alternatives):
             else:
                 return Direction()
 
+        status_codes = {
```

Comment:
this should go outside of the entire class: it's static, nothing changes. Call it `STATUS_CODES` and put it after the import